Dockerfile.tmpl

```diff
@@ -315,7 +315,7 @@ RUN pip install mpld3 && \
     pip install folium && \
     pip install scikit-plot && \
     # dipy requires the optional fury dependency for visualizations.
-    pip install fury dipy && \
+    pip install fury==0.7.1 dipy && \
```

Comment:
Could you create a bug and add the reference here as a comment for future readers?

Reply:
Already had a bug, added note.